spec/spec_helper_integration.rb

```diff
@@ -401,11 +401,11 @@ def database
     step 'add node names to puppetdb that are used in the tests' do
       node_names_file = ['file_node1', 'file_node2']
       node_names_cli  = ['nonesuch', 'andanother']
-      node_names_all  = node_names_cli + node_names_file
+      node_names_all  = node_names_cli + node_names_file + "diff_compiler"
```

Comment:
I was wondering if the string "diff_compiler" can be added to the array node_name_all by '+' operator? Or should be node_name_all.push('diff_compiler')? or array.insert or '<<' instead?